packages/core-ethereum/src/channel.ts

```diff
@@ -98,17 +98,28 @@ class Channel {
     return (await this.themToUs()).commitment
   }
 
+  getUsToThemId(): Hash {
+    return generateChannelId(this.self.toAddress(), this.counterparty.toAddress())
+  }
+
   async usToThem(): Promise<ChannelEntry> {
-    return await this.indexer.getChannel(generateChannelId(this.self.toAddress(), this.counterparty.toAddress()))
+    try {
+      return await this.indexer.getChannel(this.getUsToThemId())
+    } catch {
+      throw Error(`Channel from ${this.self.toHex()} to ${this.counterparty.toHex()} not found`)
+    }
   }
 
-  async themToUs(): Promise<ChannelEntry> {
-    return await this.indexer.getChannel(generateChannelId(this.counterparty.toAddress(), this.self.toAddress()))
+  getThemToUsId(): Hash {
```

Comment:
why does this return an id rather than the channelentry? We should change the method name then?

Reply:
we have both methods

Reply:
ah, I see, nice ok.